infrastructure/firecracker/provider.go

```diff
@@ -71,19 +68,15 @@ func (p *fcProvider) Start(ctx context.Context, vm *models.MicroVM) error {
 	return p.Create(ctx, vm)
 }
 
-// Pause will pause a started microvm.
-func (p *fcProvider) Pause(ctx context.Context, id string) error {
-	return errNotImplemeted
-}
-
-// Resume will resume a paused microvm.
-func (p *fcProvider) Resume(ctx context.Context, id string) error {
-	return errNotImplemeted
-}
-
-// Stop will stop a paused or running microvm.
+// Stop will stop a running microvm.
 func (p *fcProvider) Stop(ctx context.Context, id string) error {
```

Comment:
are we using stop anywhere?

Reply:
can we not just remove it if it serves no purpose?

Reply:
Why not. I did make me wonder if we should be doing different things in stop and delete. For example, in `Stop` should we stop the firecracker process and then in `Delete` we delete the files. But lets create a ticket for follow up and remove stop for the time being. wdyt?

Reply:
sounds good 👍

Reply:
Added #330